src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DefineAccessorsForAttributeArguments.cs

```diff
@@ -93,10 +94,11 @@ private void AnalyzeSymbol(INamedTypeSymbol symbol, INamedTypeSymbol attributeTy
             }
         }
 
-        protected abstract bool IsAssignableTo(
+        private static bool IsAssignableTo(
             [NotNullWhen(returnValue: true)] ITypeSymbol? fromSymbol,
             [NotNullWhen(returnValue: true)] ITypeSymbol? toSymbol,
-            Compilation compilation);
+            Compilation compilation)
```

Comment:
@mavasani Is there a common place for helpers like this? The code is identical in both analyzers

Reply:
[ITypeSymbolExtensions](https://github.com/dotnet/roslyn-analyzers/blob/master/src/Utilities/Compiler/Extensions/ITypeSymbolExtensions.cs) looks like a good candidate.

Reply:
@Evangelink I attempted to move it as an extension method there, but `ClassifyCommonConversion` doesn't seem to be available on public api analyzers, which references this shared project.

![image](https://user-images.githubusercontent.com/31348972/107024560-de838b80-67b0-11eb-9c4f-2dab8a34c0de.png)